changes to the grammar.md

```diff
@@ -0,0 +1,27 @@
+# Changes to the official grammar of the I language by the noname team
```

Comment:
Having spaces in the file name is considered bad luck in the programming world. Consider dashes (-)